src/lib/y2network/widgets/wireless_auth.rb

```diff
@@ -54,23 +54,31 @@ def handle(event)
 
       def contents
         Frame(
-          _("Wireless Authentication"),
+          _("Authentication"),
           VBox(
-            VSpacing(0.5),
             auth_mode_widget,
             VSpacing(0.2),
-            replace_widget,
-            VSpacing(0.5)
+            replace_widget
           )
         )
       end
 
+      # Sets the authentication mode
+      #
+      # It sets the auth mode to the given value and refreshes the widgets accordingly.
+      #
+      # @param mode [Symbol] Authentication mode
+      def auth_mode=(mode)
+        auth_mode_widget.value = mode.to_s
+        refresh
+      end
+
     private
 
       def refresh
         case auth_mode_widget.value
-        when "no-encryption" then replace_widget.replace(empty_auth_widget)
-        when "sharedkey", "open" then replace_widget.replace(wep_keys_widget)
+        when "no_encryption" then replace_widget.replace(empty_auth_widget)
```

Comment:
We have to ensure it is read properly from the ifcfg file.

Reply:
Yes, I fixed this behavior in commit 8da8620.